packages/core/src/hooks/useSendTransaction.test.ts

```diff
@@ -64,6 +64,53 @@ describe('useSendTransaction', () => {
     expect(result.current.state.errorMessage).to.eq('invalid address')
   })
 
+  it('transfer ether with limit', async () => {
```

Comment:
How are those tests testing the gas limit percentage if we're overriding the `gasLimit` to zero?

Reply:
We override gas limit to 200% of the estimated one.